doc/simulate_with_bird.md

```diff
@@ -0,0 +1,178 @@
+
+## Prerequisites
+1. A k8s cluster
+2. Ensure that the host used to simulate the router can be connected to the k8s cluster, including the bgp port and the application port in the cluster.
+3. A public IP address
+
+
+## Create a router
+
+1. Create a host in the k8s network and install Bird on the host. The QingCloud platform only has Bird 1.5 version. This version does not support ECMP. To experience all the functions of Porter, you need to install at least 1.6 version. Execute the following script to install Bird 1.6.
+    ```
+     $sudo add-apt-repository ppa:cz.nic-labs/bird
+     $sudo apt-get update 
+     $sudo apt-get install bird
+     $sudo systemctl enable bird  
+    ```
+
+2. Configure the router's BGP service. Modify `/etc/bird/bird.conf` and add the following parameters:
+    ```
+    protocol bgp mymaster {   
+        description "192.168.1.4";  # Router ID, usually the main IP address
+        local as 65001;             # Local AS number, must be different from the AS number of the k8s cluster
+        neighbor 192.168.1.5 port 17900 as 65000;  # Master node IP and AS number
+        source address 192.168.1.4;                # Router IP  
+        import all;   
+        export all;
+        enable route refresh off;  # Due to the low BGP protocol of bird 1.6, multiple routes advertised by Porter will become a single route, this parameter can be used as a workaround to fix this problem.
+        add paths on; # When this parameter is set to on, you can receive multiple routes from the Porter.
+    }
+     ```
+   The above parameters configure a neighbor to the simulated router. The neighbor is the master node of the cluster. **We assume that your Porter controller is deployed on the master node. If you do not want to restrict the porter to be deployed on the master node or the master cannot deploy pods, then you need to add all neighbor nodes to this configuration file according to the above rules.** Modify the `kernel` part of the file, cancel the `export all` comments, and enable the ECMP function:
+   ```
+   protocol kernel {
+        scan time 60;
+        import none;
+        export all;     # Actually insert routes into the kernel routing table
+        merge paths on; # Enable ECMP, this parameter requires at least bird 1.6
+   }
+
+   ```
+
+3. Reboot Bird
+   ```bash
+    $sudo systemctl restart bird 
+   ```
+
+4. Configure the public IP. Apply for an **internal binding** public network IP on the QingCloud console. Note that it must be an internal binding IP. If it is external binding, the public IP cannot be found. Bind this IP to the host where the simulated router is located. You only need to complete the binding. There is no need to follow up with the QingCloud documentation.
```

Comment:
Configure the Elastic IP. Apply an Elastic IP that **Associate Mode
is Internal** on the QingCloud console. Please note that EIP must be an internal associate mode. If it is an external associate mode, EIP cannot be found in VM. Associate this EIP to the VM where the simulated router is located. You only need to complete the associate. There is no need to follow up with the QingCloud documentation.